pyproject.toml

```diff
@@ -64,6 +64,8 @@ db-dtypes = { version = ">=0.3,<2", optional = true }
 deltalake = { version = ">=0.9.0,<1", optional = true }
 duckdb = { version = ">=0.8.1,<2", optional = true }
 geopandas = { version = ">=0.6,<2", optional = true }
+geoarrow-types = { version = ">=0.2,<1", optional = true }
+pyproj = { version = ">=3.3.0,<4", optional = true }
```

Comment:
Just to be sure, we are making `pyproj` an explicit dependency. I know it comes when installing geopandas, which is already a dependency. I guess explicit better than implicit in this case.

Reply:
Yeah, it's generally a bad idea to assume that a package will forever and always include a dependency that you require. If there's a top-level import for a package, it needs to be included as an explicitly dependency.